core-bundle/src/Resources/contao/classes/Ajax.php

```diff
@@ -328,7 +329,21 @@ public function executePostActions(DataContainer $dc)
 
 				// Set the new value
 				$varValue = Input::post('value', true);
-				$strKey = ($this->strAction == 'reloadPagetree') ? 'pageTree' : 'fileTree';
+
+				switch ($this->strAction)
+				{
+					case 'reloadPicker':
+						$strKey = 'picker';
+						break;
+
+					case 'reloadPagetree':
+						$strKey = 'pageTree';
+						break;
+
+					case 'reloadFiletree':
```

Comment:
No need for it as it fallbacks to default anyway.

Reply:
I like it to be more explicit. One can read in the code what other case there is.